provision/acc_provision/apic_provision.py

```diff
@@ -1933,6 +1934,136 @@ def nested_dom(self):
         self.annotateApicObjects(data)
         return path, data
 
+    def nested_dom_second_portgroup(self):
```

Comment:
As a follow up we can perhaps consider refactoring the vmmUsrCustomAggr into one method so that we can call it from two different places and avoid some repetition of code.